<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ssoi_port_widget</class>
 <widget class="QWidget" name="ssoi_port_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>56</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="SsoiM_PortNum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>27</y>
     <width>106</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>66</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>999</width>
     <height>999</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_82">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>COM-Порт</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>1</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="OprosVariant_1">
    <widget class="QLabel" name="label_84">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Запросы</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_83">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Интервал,мс</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="SsoiM_Interval">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>26</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>3000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="SsoiM_MaxErrCnt">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>26</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="OprosVariant_3">
    <widget class="QLabel" name="label_85">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Интервал опроса, мс</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="SsoiM_Interval_">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>26</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="OprosVariant_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
